src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs

```diff
@@ -802,6 +813,66 @@ public static void TestingGetStringInvalidUTF8(byte[] dataUtf8)
             }
         }
 
+        [Theory]
+        [MemberData(nameof(GetCommentTestData))]
+        public static void TestingGetComment(string jsonData, string expected)
+        {
+            byte[] dataUtf8 = Encoding.UTF8.GetBytes(jsonData);
+            var state = new JsonReaderState(options: new JsonReaderOptions { CommentHandling = JsonCommentHandling.Allow });
+            var reader = new Utf8JsonReader(dataUtf8, isFinalBlock: true, state);
+            bool commentFound = false;
+            bool startObjectFound = false;
+            bool endObjectFound = false;
+            while (reader.Read())
+            {
+                switch (reader.TokenType)
+                {
+                    case JsonTokenType.StartObject:
+                        startObjectFound = true;
+                        break;
+                    case JsonTokenType.EndObject:
+                        endObjectFound = true;
+                        break;
+                    case JsonTokenType.Comment:
+                        commentFound = true;
+                        Assert.Equal(expected, reader.GetComment());
```

Comment:
nit: Add `Assert.True(reader.TryGetComment());`

Reply:
Nevermind, I just realized we are not adding a `Try*` method (similar to how we don't have `TryGetString`).

Reply:
nit: Let's simplify this test as well given the input is always `{ comment }`. We don't need the `*found` flags (especially when you have default case which fails the test).

```C#
Assert.True(reader.Read());
Assert.Equal(JsonTokenType.StartObject, reader.TokenType);

Assert.True(reader.Read());
Assert.Equal(JsonTokenType.Comment, reader.TokenType);
Assert.Equal(expected, reader.GetComment());

Assert.True(reader.Read());
Assert.Equal(JsonTokenType.EndObject, reader.TokenType);

Assert.False(reader.Read());
```

Reply:
Done.